system_tests/bigquery.py

```diff
@@ -602,3 +613,18 @@ def test_dump_table_w_public_data(self):
         dataset = Config.CLIENT.dataset(DATASET_NAME, project=PUBLIC)
         table = dataset.table(TABLE_NAME)
         self._fetch_single_page(table)
+
+    def test_create_table_nested_schema(self):
+        TABLE_NAME = 'test_table'
```

Comment:
I don't understand how this is "not a constant" -- it is a semantically-named label for what would otherwise be a "magic" literal.  Is it because we aren't defining it at module scope?  That would be a silly requirement to start enforcing:  we have literally thousands of examples of non-module-scope, all-caps variable assignments in 164 modules scattered throughout our code.